scripts/qa_crowbarsetup.sh

```diff
@@ -504,18 +504,19 @@ function resize_partition()
 
 function export_tftpboot_repos_dir()
 {
-    tftpboot_repos_dir=/srv/tftpboot/repos
-    tftpboot_suse_dir=/srv/tftpboot/suse-11.3
-
-    if iscloudver 5; then
+    if iscloudver 7plus; then
+        tftpboot_suse12sp2_dir=/srv/tftpboot/suse-12.2
+        tftpboot_repos12sp2_dir=$tftpboot_suse12sp2_dir/x86_64/repos
```

Comment:
I believe we need to set also`tftpboot_suse12sp1_dir` here, it is used from ` onadmin_prepare_sles12sp1_repo`